src/components/Profile/index.js

```diff
@@ -12,7 +12,7 @@ const Profile = () => {
 
   const navigate = useNavigate();
 
-  const logoutHandler = (e) => {
+  const logoutHandler = (e, navigate) => {
     e.preventDefault();
```

Comment:
Suggestion: you could use more specific name  instead of e. for example logoutEvent